src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -212,10 +212,20 @@ private void setupKeyboardShortcuts() {
             if (NEW_COMMENT.match(event)) {
                 if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
                     ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
-                } else if (ui.getBrowserComponent().isCurrentUrlIssue()) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
-                    ui.getBrowserComponent().jumpToComment();
+                } else {
+                    if (!ui.getBrowserComponent().isCurrentUrlPrDiscussion()) {
+                        ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                    }
+
+                    Thread jumpToCommentOncePageLoaded = new Thread(() -> {
+                        while (!ui.getBrowserComponent().isCurrentUrlPrDiscussion()) {
+                        // wait
+                        }
+                        ui.getBrowserComponent().jumpToComment();
+                    });
+                    jumpToCommentOncePageLoaded.start();
```

Comment:
@acruis will this be better or would `thread.sleep()` be better? I am thinking of using shortcut to load webpages in quick succession, in which using `thread.sleep` would be better.

Something like:
```
while (//page not loaded) {
  Thread.sleep((100); // low value to not affect responsiveness
}
```